MODULE.bazel

```diff
@@ -104,26 +103,26 @@ local_path_override(
 
 git_override(
     module_name = "bazel_remote_apis",
-    commit = "7f922028fcfac63bdd8431e68de152d9e7a9e2a0",  # bb-storage
-    # commit = "a6328f5026d3b2ae27bd8efd86a0f1057ccab361",  # bb-remote-execution
+    commit = "9ef19c6b5fbf77d6dd9d84d75fbb5a20a6b62ef1",  # bb-storage
+    # commit = "e94a7ece2a1e8da1dcf278a0baf2edfe7baafb94",  # bb-remote-execution
     remote = "https://github.com/bazelbuild/remote-apis.git",
 )
 
 git_override(
     module_name = "com_github_buildbarn_bb_browser",
-    commit = "9b44996f7d6a04185f61e7752cbe8649aaa59f16",
+    commit = "5d2ed98e07561fdad8b50d995e9b1a76d4a96450",
     remote = "https://github.com/buildbarn/bb-browser.git",
 )
 
 git_override(
     module_name = "com_github_buildbarn_bb_remote_execution",
-    commit = "1c726bdc27e7793c685d8788913f8f91f59bc887",
+    commit = "135832aa47c42781f2af698a4823beda0863bad3",
```

Comment:
Please upgrade to https://github.com/buildbarn/bb-remote-execution/commit/efef252cdf4f97cc0154ea02976ed1322587e599.